test/e2e/common/empty_dir.go

```diff
@@ -207,7 +207,13 @@ var _ = Describe("[sig-storage] EmptyDir volumes", func() {
 		doTest0777(f, testImageNonRootUid, v1.StorageMediumDefault)
 	})
 
-	It("pod should support shared volumes between containers", func() {
+	/*
+		Release : v1.15
+		Testname: EmptyDir, Shared volumes between containers
```

Comment:
Sorry if I'm dense here, but what is EmptyDir on windows? 

/assign @timothysc

Reply:
I don't quite understand your question. An EmptyDir is an empty dir. The only other thing would be if the ``Medium=memory`` is set, but that is ignored by Windows Kubelet.

Reply:
If it's ignored by windows shouldn't it be [LinuxOnly]

Reply:
@kubernetes/sig-windows-misc

Reply:
if set `Medium=memory` for emptyDir on Windows, it will create a file actually:
https://github.com/kubernetes/kubernetes/blob/d24fe8a801748953a5c34fd34faa8005c6ad1770/pkg/util/mount/mount_windows.go#L60-L63
Does this case work on Windows?

Reply:
Memory as a storage medium is not supported on Windows.  I am not sure what will happen in that case, but it will likely not work.

Reply:
As I said, the ``Medium=memory`` doesn't really do anything Windows and a regular EmptyDir is created: https://paste.ubuntu.com/p/JMcp8g58Xk/ . This fact can also be seen in @andyzhangx 's comment, it can be seen that a regular folder is created if the source is memory / tmpfs.

It could arguably be considered a bug, since it doesn't have the same behaviour as on Linux (the volume is not stored in memory, nor will it be deleted on reboot).

Reply:
This test case is not using EmptyDir as medium=memory so I don't think this memory issue should block this test.

Reply:
Can someone run this test case on a windows platform to see if any changes to this test need to be made?

Reply:
I did, and it works. I've already left a general comment about this yesterday.